google/cloud/internal/oauth2_minimal_iam_credentials_rest_test.cc

```diff
@@ -253,6 +255,11 @@ TEST(MinimalIamCredentialsRestTest, GenerateAccessTokenSuccess) {
   EXPECT_CALL(*mock_credentials, GetToken).WillOnce([lifetime](auto tp) {
     return AccessToken{"test-token", tp + lifetime};
   });
+  EXPECT_CALL(*mock_credentials, universe_domain)
+      .WillOnce([&](Options const&) -> StatusOr<std::string> {
+        if (ud) return *ud;
```

Comment:
my opinion: we should avoid conditionals like this in tests to keep them as simple as possible

Reply:
Done.